File: 프로그래머스/0/120843. 공 던지기/공 던지기.cpp
#include <string>
#include <vector>

using namespace std;

int solution(vector<int> numbers, int k) {
    int answer = 0, index = 0;

    for (int i = 1; i <= k; i++) {
        answer = numbers[index];
        index = (index + 2) % numbers.size();
    }

    return answer;
}

File: 프로그래머스/0/120843. 공 던지기/공 던지기.swift
import Foundation

func solution(_ numbers:[Int], _ k:Int) -> Int {
    
    var index = 0
    var cnt = 0
    
    while true {
        cnt += 1
        if cnt == k { break }
        index = (index + 2) % numbers.count
    }
    
    return numbers[index]
}